src/components/Spacer/Spacer.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+
+import classNames from 'classnames';
+
+export interface SpacerProps {
+	side?: 'left' | 'top' | 'right' | 'bottom';
```

Comment:
helaas kan je 2 opties toevoegen aan 1 div
```
<div class="u-spacer-top u-spacer-bottom">
```

dus we moeten een array van strings aanvaarden, maar mss ook een single string.
```
side?: ('left' | 'top' | 'right' | 'bottom')[],
size?: ('large' | 'medium' | 'small')[];
```